src/app/frontend/_variables.scss

```diff
@@ -31,6 +31,7 @@ $primary: #326de6;
 $secondary: #f51200;
 $warning: #ffa500;
 $delicate: #aaa;
+$border: #ddd;
```

Comment:
Isn't border the same as foreground-4? How about using foreground-4 color instead of $border? General idea is the less variables the better.